Python/jit.c

```diff
@@ -421,11 +423,7 @@ _PyJIT_Compile(_PyExecutorObject *executor, const _PyUOpInstruction *trace, size
     // Loop again to emit the code:
     unsigned char *code = memory;
     unsigned char *data = memory + code_size;
-    unsigned char *top = code;
-    if (trace[0].opcode == _START_EXECUTOR) {
-        // Don't want to execute this more than once:
-        top += stencil_groups[_START_EXECUTOR].code.body_size;
-    }
+    assert(trace[0].opcode == _START_EXECUTOR);
```

Comment:
I thought the first opcode could still be `_COLD_EXIT`? Or is that gone now? Also this assert is failing on the debug builds.

Reply:
Oh I guess it can't be cold exit anymore. Hmm strange that it's still failing then.